eslint.config.js

```diff
@@ -0,0 +1,45 @@
+import globals from 'globals';
+import eslint from '@eslint/js';
+import typescriptParser from '@typescript-eslint/parser';
+import mochaPlugin from 'eslint-plugin-mocha';
+import reactPlugin from 'eslint-plugin-react';
+
+const GLOBALS_BROWSER_FIX = Object.assign({}, globals.browser, {
+  AudioWorkletGlobalScope: globals.browser['AudioWorkletGlobalScope '],
+});
+
+delete GLOBALS_BROWSER_FIX['AudioWorkletGlobalScope '];
+
+export default [
+  eslint.configs.recommended,
+  { ignores: ['dist', 'docs'] },
+  {
+    languageOptions: {
+      ecmaVersion: 'latest',
+      sourceType: 'module',
+      parser: typescriptParser,
+      globals: {
+        ...globals.node,
+        ...globals.mocha,
+        ...GLOBALS_BROWSER_FIX,
```

Comment:
Our dependency `@babel/plugin-transform-private-property-in-object` relies on a version of `globals` where the `AudioWorkletGlobalScope` property in `globals.browser` contains an extra trailing space, causing a linting error.

As a temporary fix, I addressed this by removing the extra space (GLOBALS_BROWSER_FIX).

Alternatively, we could consider overriding the globals package in `package.json`:

```json
"overrides": {
  "globals": "^15.12.0"
}
```